script/results.html

```diff
@@ -215,30 +440,290 @@
+            for (const key in input) {
+                for (const item of input[key]) {
+                    if (item.state == 'passed') {
+                        tests_total++
+                        tests_passed++
+                    } else if (item.state == 'skipped') {
+                        tests_total++
+                        tests_skipped++
+                    } else if (item.state == 'failed') {
+                        tests_total++
+                        tests_failed++
+                    }
                 }
-                text += '</tbody></table>'
+            }
+
+            // First we will have a left column in the table with the summarized results
+            text ='<thead><tr><th scope="col">Test summary</th><th scope="col">Test feedback</th></tr></thead><tbody>'
+            text += '<tr><td class="align-top"><b>Total:</b> ' + tests_total + '<br><b>Passed:</b> ' + tests_passed + '<br><b>Skipped:</b> ' + tests_skipped + '<br><b>Failed:</b> ' + tests_failed + '</td><td>'
+            text += '<div class="accordion" id="feedback-accordion">'
+
+            for (const key in input) {
+              for (const key2 in input2){      
+              
+              eqData=input2[key2][0].FarEdge
+            
+            if (tableName == "Telco"){
+              eqData=input2[key2][0].ForTelco
+
+            }
+            if (tableName == "NonTelco"){
+              eqData=input2[key2][0].ForNonTelco
+            }
+
+            if (key==key2 && eqData=="Mandatory"){
+                // NOTE: we are assuming the test result is determined by the passed/failed state of the first item
+                status = input[key][0].state
+                if (status === "passed") {
+                    buttontype = 'bg-success text-white'
+                } else if (status === "skipped") {
+                    buttontype = 'bg-warning text-black'
+                } else {
+                    buttontype = 'bg-danger text-white'
+                }   
+
+                itemid = 'collapse' + id
+                headingid = 'heading' + id
+                text += '<div class="accordion-item"><h2 class="accordion-header" id="' + headingid + '"><button class="accordion-button collapsed ' + buttontype + '" type="button" data-bs-toggle="collapse" data-bs-target="#' + itemid + '" aria-expanded="true" aria-controls="' + itemid +'">'
+                text += key + '</button></h2>'
+                // Now we should populate the item contents
+                text += '<div id="' + itemid + '" class="accordion-collapse collapse" aria-labelledby="' + headingid + '" data-bs-parent="' + element + '">'
+                text += '<div class="accordion-body">'
+                // Inside the accordion, 1 table with the following colummns
+
+                for (const item of input[key]) {
+                testName = item.testID.url.split('http://test-network-function.com/testcases/')[1].replaceAll('/', '-')
```

Comment:
I could not load the claim.json, the split base on the url would not work anymore since url was removed recently